inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/listener/RestartSortListener.java

```diff
@@ -98,21 +95,6 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
             return ListenerResult.fail(message);
         }
 
-        // TODO Support more than one dataflow in one sort job
```

Comment:
@yunqingmoswu @kipshi 
Do we already support one Sort task to write data to multiple sinks at the same time?

Reply:
Yes, manager already support this before.